### PR TITLE
web: add a warning when there are no webhook logs.

### PR DESCRIPTION
**Pull request by sashaostrikov**
Test plan:
Storybook.

Closes https://github.com/sourcegraph/sourcegraph/issues/45325

## App preview:

- [Web](https://sg-web-ao-ui-webhooks-no-logs-warning.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | 0.00% (+0.26 kb) | 0.00% (+0.26 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/ao/ui/webhooks/no-logs-warning/compare-report.html;tab=live_object) for a full comparison between the commits 8cfdc65 and 8ef72ea or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

File: client/web/src/site-admin/SiteAdminWebhookPage.tsx [hunk 1]
```diff
@@ -131,6 +142,13 @@ export const SiteAdminWebhookPage: FC<WebhookPageProps> = props => {
                 />
             )}
 
+            {connection?.nodes?.length === 0 && (
+                <Alert className="mt-2" variant="warning">
+                    <H4>Webhook receiver created</H4>
+                    You must create a webhook on the external code host using the endpoint below to enable this
+                    functionality.
+                </Alert>
+            )}
```
Comment (mrnugget) [inline client/web/src/site-admin/SiteAdminWebhookPage.tsx:151]: I don't think this should be an alert, to be honest. It's completely expected that there might not be requests yet.

(also: the header here is wrong).


Why not just have an "empty connection" thing down in the `ConnectionContainer`? Like this one here:

screenshot_2022-12-12_13 17 58@2x

We could just add normal text like this and have it say something like

> No requests received yet. Be sure to configure the webhook on the code host.

And make "configure the webhook on the code host" a link to `/help/admin/config/webhooks#configuring-webhooks-on-the-code-host` (which redirects from the instance to `docs.sourcegraph.com/admin/config/webhooks#configuring-webhooks-on-the-code-host`)

Comment (sashaostrikov) [reply client/web/src/site-admin/SiteAdminWebhookPage.tsx:151]: I believe main intention behind an alert was that we shouldn't have webhooks without logs (why need a webhook without events). But I agree on explaining it as an empty log message.

Comment (mrnugget) [reply client/web/src/site-admin/SiteAdminWebhookPage.tsx:151]: Yeah, but if you're a user who creates a new webhook it's weird that you're greeted with a warning for something that's expected, right? Our docs tell you to go create a webhook and _then_ configure it on the code host. We shouldn't warn you about it.

Comment (sashaostrikov) [reply client/web/src/site-admin/SiteAdminWebhookPage.tsx:151]: Makes sense, will change it to inlined message and remove the alert 👍🏻

**Comment by sashaostrikov** [comment]
Screenshot from local sg
image

File: client/web/src/site-admin/SiteAdminWebhookPage.tsx [hunk 2]
```diff
@@ -182,4 +182,12 @@ const Header: FC = () => (
     </>
 )
 
-const EmptyList: FC = () => <div className="m-4 w-100 text-center">No webhook logs found</div>
+const EmptyList: FC = () => (
+    <div className="m-4 w-100 text-center">
```
Comment (mrnugget) [inline client/web/src/site-admin/SiteAdminWebhookPage.tsx:186]:
```suggestion
    <div className="m-4 w-100 text-center text-muted">
```

Like here: https://github.com/sourcegraph/sourcegraph/blob/bd479deb380645b29bed678a9f78e70b7e4db669/client/web/src/savedSearches/SavedSearchListPage.tsx#L205

**Review by mrnugget — APPROVED** [review]

**Pull request merged**